Tests/ApolloTests/ApolloClientOperationTests.swift

```diff
@@ -41,21 +41,59 @@ final class ApolloClientOperationTests: XCTestCase {
     }
   }
 
-  func test__performMutation_givenPublishResultToStore_false_doesNotPublishResultsToStore() throws {
-    // given
-    class GivenSelectionSet: MockSelectionSet {
+  // given
+  class GivenSelectionSet: MockSelectionSet {
+    override class var __selections: [Selection] { [
+      .field("createReview", CreateReview.self)
+    ] }
+
+    class CreateReview: MockSelectionSet {
       override class var __selections: [Selection] { [
-        .field("createReview", CreateReview.self)
+        .field("__typename", String.self),
+        .field("stars", Int.self),
+        .field("commentary", String?.self)
       ] }
+    }
+  }
 
-      class CreateReview: MockSelectionSet {
-        override class var __selections: [Selection] { [
-          .field("__typename", String.self),
-          .field("stars", Int.self),
-          .field("commentary", String?.self)
-        ] }
-      }
+  func test__performMutation_givenPublishResultToStore_true_publishResultsToStore() throws {
+    let mutation = MockMutation<GivenSelectionSet>()
+    let resultObserver = self.makeResultObserver(for: mutation)
+
+    let serverRequestExpectation = self.server.expect(MockMutation<GivenSelectionSet>.self) { _ in
+      [
```

Comment:
This looks like common data between both tests and could be pulled out into the test class.

Reply:
Indeed.
fixed: [92c233a](https://github.com/apollographql/apollo-ios/pull/2953/commits/92c233a940ad1236e2c9d6e71055e86e1a30e2ba)